app/dashboard/views.py

```diff
@@ -4518,18 +4532,19 @@ def get_project(request, project_id):
                 'url': member_profile.url,
                 'handle': member_profile.handle,
                 'avatar': member_profile.avatar_url
-            } for member_profile in project.profiles.all()]
+            } for member_profile in project.profiles.all()],
+            'team_members_profile': [member_profile for member_profile in project.profiles.all()]
         },
         'hackathon': hackathon_obj[0],
     }
+    return params
```

Comment:
Why isn't this a JsonResponse ?

Reply:
Using this as an internal function to share project details. The JsonResponse is returned by the function above this one **get_project()**.